core/color.cpp

```diff
@@ -526,12 +526,12 @@ Color Color::operator/(const Color &p_color) const {
 			a / p_color.a);
 }
 
-Color Color::operator/(const real_t &rvalue) const {
+Color Color::operator/(real_t p_rvalue) const {
 	return Color(
-			r / rvalue,
-			g / rvalue,
-			b / rvalue,
-			a / rvalue);
+			r / p_rvalue,
+			g / p_rvalue,
+			b / p_rvalue,
+			a / p_rvalue);
```

Comment:
Why `p_rvalue` over `p_value`?

Given that we already use the `r_value` convention for return values, this has the potential for confusion imo, and it would be better to avoid `rvalue`.

Reply:
Did I not push those changes? I changed all of those to `p_scalar`, will check